packages/expo-modules-core/ios/ExpoModulesCore.h

```diff
@@ -1,5 +1,94 @@
-// Copyright 2015-present 650 Industries. All rights reserved.
+@class RCTBundleURLProvider;
 
-#if __has_include(<ExpoModulesCore/ExpoModulesCore-umbrella.h>)
-#import <ExpoModulesCore/ExpoModulesCore-umbrella.h>
-#endif
+#import <React/RCTHTTPRequestHandler.h>
+
+#import "ExpoModulesCore/ExpoBridgeModule.h"
```

Comment:
```suggestion
#import <ExpoModulesCore/ExpoBridgeModule.h>
```

use framework header style